src/containers/CandidateListPage/TestList/index.js

```diff
@@ -122,17 +105,7 @@ class TestList extends React.Component {
                     placement="top"
                     title="write summary"
                     onClick={this.handleSummaryEdit}
-                  >
```

Comment:
here we still need the atleastoneendrecord to check if the test is still going on(it require at least one record to make the"open summary" button visible, other wise it will be "ongoing"